test/regress/run_regression.py

```diff
@@ -44,13 +44,13 @@ def __init__(self):
     def applies(self, benchmark_info):
         return True
 
-    def run(self, benchmark_info):
+    def run(self, benchmark_info, compare_outputs=True):
```

Comment:
It would be more consistent with the rest of the code to make `compare_outputs` part of `benchmark_info` IMHO. What do you think?

Reply:
I agree. Though, I wonder if we should rename `benchmark_info` to `benchamrk_config` (or something else) as `comapre_outputs` is specific to the testers and not part of the benchmark.

Reply:
Yes, `benchmark_config` sounds pretty good. We can do that in an upcoming PR.